specs/model-avro.spec/src/main/scripts/io/aklivity/zilla/specs/model/avro/config/event.yaml

```diff
@@ -23,7 +23,8 @@ telemetry:
         events:
           - qname: test.net0
             id: model.avro.validation.failed
-            message: VALIDATION_FAILED java.io.EOFException
+            name: MODEL_AVRO_VALIDATION_FAILED
+            message: A message payload failed validation. java.io.EOFException.
```

Comment:
Please remove exception class name from message.

Reply:
`"java.io.EOFException"` is one of the exception messages returned from the `AvroRuntimeException` class. The `error` string passed in the event is set from various exception messages. 

I think this is just a bad example of what kind of errors might come through and I don't know how the tests makes this specific exception happen or how to change it to something else.

origional comment here: https://github.com/aklivity/zilla/pull/1029#discussion_r1624780061

Reply:
I also took fixing this as a not in some follow on modifications that are needed: https://github.com/aklivity/zilla/pull/1029#issuecomment-2145701938